suite-native/forms/package.json

```diff
@@ -13,7 +13,7 @@
     "dependencies": {
         "@suite-native/atoms": "*",
         "react": "18.2.0",
-        "react-hook-form": "6.15.7",
+        "react-hook-form": "7.39.4",
```

Comment:
I will update suite-native part once you are done with desktop but should be quick, we have react-hook-form abstracted so we do not access API directly